easybuild/easyconfigs/d/deepdiff/deepdiff-3.3.0-intel-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'PythonPackage'
+
+name = 'deepdiff'
+version = '3.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://deepdiff.readthedocs.io/en/latest/'
+description = """DeepDiff: Deep Difference of dictionaries, iterables and almost any other object recursively."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['ecad8e16a96ffd27e8f40c9801a6ab16ec6a7e7e6e6859a7710ba4695f22702c']
+
+dependencies = [('Python', '2.7.15')]
+
```

Comment:
@hajgato Please set both `download_dep_fail = True` and `use_pip = True` in easyconfigs using `PythonPackage`